src/components/forms/EditExerciseConfigForm/EditExerciseConfigVariable.js

```diff
@@ -65,12 +63,10 @@ const EditExerciseConfigVariable = ({ prefix, data, supplementaryFiles }) =>
                 options={[{ key: '', name: '...' }].concat(
                   supplementaryFiles
                     .sort((a, b) => a.name.localeCompare(b.name))
-                    .map(data => {
-                      const obj = {};
-                      obj['key'] = data.hashName;
-                      obj['name'] = data.name;
-                      return obj;
-                    })
+                    .map((data, i) => ({
+                      key: i,
```

Comment:
here is no `hashName` ... it really works this way?

Reply:
Damn, you are right - but there is a different problem with the keys then. The console is full of warnings here - we should instead do the following:

- filter out duplicities
- return back `key: data.hashName,`
- do this for all of the selects in this file

Could you do that?

Reply:
Yes, user can upload one file multiple times, that's the warnings I didn't see because my job config is clean (no duplicities). I'll fix this :)